### PR TITLE
Update to Hadoop and CDH tables

### PR DESCRIPTION
**Pull request by awholegunch**
Updates tables in the Admin Manual.

Running as Quick Build: http://builds.cask.co/browse/CDAP-DQB196-2

Pages of interest: 
- http://builds.cask.co/artifact/CDAP-DQB196/shared/build-2/Docs-HTML/4.0.0-SNAPSHOT/en/admin-manual/hadoop-compatibility.html
- http://builds.cask.co/artifact/CDAP-DQB196/shared/build-2/Docs-HTML/4.0.0-SNAPSHOT/en/admin-manual/installation/cloudera.html#downloading-and-installing-csd

### CODE REVIEW

File: cdap-docs/admin-manual/source/installation/cloudera.rst [hunk 1]
```diff
@@ -122,6 +122,10 @@ available services which CM can install.
 +------------+-----------------------+--------------------------------------+
 | CM Version | CDH Version           | CDAP Parcel / CSD Version            |
 +============+=======================+======================================+
+| 5.9        | 5.9.x                 | |short-version-x|                    |
```
Comment (dereklwood) [inline cdap-docs/admin-manual/source/installation/cloudera.rst:125]: we are missing the older CDH/CDAP versions for this (CM5.9), and CM 5.8 below.  We need to look up if there were any compatibility breaks between CDAP and CDH 5.7+ first

File: cdap-docs/admin-manual/source/hadoop-compatibility.rst [hunk 2]
```diff
@@ -61,23 +61,23 @@ For a Distributed CDAP cluster, version |release|, you **must** install these Ha
 +                +---------------------------------+---------------------------------------+
```
Comment (dereklwood) [inline cdap-docs/admin-manual/source/hadoop-compatibility.rst:61]: few lines up: Apache HBase 1.2, per https://issues.cask.co/browse/CDAP-6937

File: cdap-docs/admin-manual/source/installation/cloudera.rst (hunk 1 above)
Comment (dereklwood) [reply cdap-docs/admin-manual/source/installation/cloudera.rst:125]: CDAP 3.5  (technically 3.5.2, 3.6.1) and up support CDH 5.8 and 5.9, per https://issues.cask.co/browse/CDAP-6234 and https://issues.cask.co/browse/CDAP-7291

Comment (wolf31o2) [reply cdap-docs/admin-manual/source/installation/cloudera.rst:125]: Well, 3.5.2 and 3.6.1 aren't released, yet.

File: cdap-docs/admin-manual/source/hadoop-compatibility.rst (hunk 2 above)
Comment (awholegunch) [reply cdap-docs/admin-manual/source/hadoop-compatibility.rst:61]: Fixed.

**Comment by awholegunch** [comment]
"On-hold" as we need to make changes.

**Comment by dereklwood** [comment]
Pushed a commit, please doublecheck

**Comment by awholegunch** [comment]
Running a new build.

**Comment by dereklwood** [comment]
👍 lgtm

**Pull request merged**